packages/tenant-process/src/model/domain/errors.ts

```diff
@@ -163,3 +164,11 @@ export function mailNotFound(mailId: string): ApiError<ErrorCodes> {
     title: "Mail not found",
   });
 }
+
+export function mailAlreadyExists(address: string): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `mail with address ${address} already exists`,
```

Comment:
address is a potential sensitive info, we should avoid adding where it could be logged
```suggestion
    detail: `mail already exists`,
```